### DISCUSSION TITLE
development splitChunks

### DISCUSSION THREAD

**Discussion by 273403986** [gatsbyjs/gatsby#26865] [Help] [answered]
How does development mode enable the same code segmentation as production mode?

**Comment by LekoArts** [comment]
Please explain your question in more detail, I don't understand what you're asking for. Thanks!

**Comment by 273403986** [comment]
In the development environment, I found that the request package CommonJS is particularly large, can it be divided with the same code in the production environment to reduce the size of the package volume.

**Comment by LekoArts** [comment]
No, that's not a built-in option and also not necessary as the development version is not optimized on purpose (to allow quicker development). The `gatsby develop` process shouldn't be used for hosting the site of course.

**Comment by 273403986** [comment]
Then how can the current page not be affected during the gatsby packing build process? Since Gatsby build will clear the current. Cache and public first, What I want is that when I repackage it, it will not affect the current users.

**Comment by LekoArts** [comment, marked as answer]
If you're not using any of these services (https://www.gatsbyjs.com/docs/deploying-and-hosting/) and are self-hosting you'll need to create the build in another folder and then once finished copy the files over to the desired folder for your hosting/website.
